adapter.go

```diff
@@ -49,6 +49,8 @@ type CasbinRule struct {
 	V3    string `gorm:"size:100"`
 	V4    string `gorm:"size:100"`
 	V5    string `gorm:"size:100"`
+	V6    string `gorm:"size:25"`
```

Comment:
size:100 instead of 25 to be consistent

Reply:
Set the size of V6 and V7 as 100 will make the length of unique index exceed the mysql default limitation, which is 3072 bytes.

Reply:
@tangyang9464 using 100 will cause index overflow. So he uses 25 here as a workaround.